decidim-proposals/spec/system/collaborative_drafts_spec.rb

```diff
@@ -94,7 +94,7 @@
       end
 
       it "shows the state" do
-        expect(page).to have_css(".label.collaborative-draft-status", text: translated(collaborative_draft.state))
+        expect(page).to have_css(".label.collaborative-draft-status", text: "Open")
```

Comment:
A little question here @mrcasals , why is this done in reverse? I mean, instead of using "translated" method, here is used hardoced value?

Reply:
If I left it with the `translated` method it complained it couldn't find the text (it was looking for `"open"` but found `"Open"`, note the capitalization).